test/WebSites/Basic/Swagger/AddCartDefault.cs

```diff
@@ -17,9 +18,9 @@ public void Apply(Schema model, ModelFilterContext context)
         {
             _logger.LogInformation("Applying an awesome IModelFilter that leverages Dependency Injection");
 
-            model.Default = new
+            model.Default = new Cart
             {
-                Id = "myCartId"
+                Id = 123
```

Comment:
Right now - anonymous types are better for setting examples because the Swagger-specific serializer may not reflect the API's serialization settings (camelCasing etc.). At least with the anonymous types, you can control the casing yourself.